deploy/crds/aws_v1alpha1_accountclaim_5_cr.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: aws.managed.openshift.io/v1alpha1
+kind: AccountClaim
+metadata:
+  name: corp-org
+  namespace: corp-org-ns
+spec:
+  legalEntity:
+    name: Red Hat OpenShift Online
+    id: 1HELaFOf2YHWwwvt3XMbT5Mja7M
```

Comment:
@rogbas this is dummy data right?

Reply:
yes, it's dummy. I just updated it with a more generic legal entity